nix/haskell-pins.nix

```diff
@@ -266,6 +266,15 @@ let
         "warp" = "warp";
       };
     };
+
+    HsOpenSSL = {
+      src = fetchgit {
+        url = "https://github.com/wireapp/HsOpenSSL";
+        rev = "2d7afef71ecdc2edd579dde072b75674da1fb730";
+        sha256 = "sha256-wLjDOMokKELVH30bYNyDAYT3ux3pdMq4sIeMZLQOSG8=";
+      };
+    };
+
```

Comment:
There's been a new release of HsOpenSSL already with this! So, this could be moved to hackagePins.

https://github.com/haskell-cryptography/HsOpenSSL/pull/91#issuecomment-2161667646